lib/posix-process/Config.uk

```diff
@@ -4,50 +4,44 @@ menuconfig LIBPOSIX_PROCESS
 
 if LIBPOSIX_PROCESS
 
-menuconfig LIBPOSIX_PROCESS_PIDS
```

Comment:
Commit meassage headline for 
 lib/posix-process: Refactor Kconfig options for multithreading/multip…

…rocess

is too long and spills into the next line

I know you added a Checkpatch-Ignore, but I'd rather not ahve it like this in our upstream history.